client/src/style/scss/base.scss

```diff
@@ -410,13 +411,18 @@ $galaxy-state-bg: (
     "error": $state-danger-bg,
     "deleted": darken($state-default-bg, 30%),
     "hidden": $state-default-bg,
+    "setting_metadata": $state-warning-bg,
 );
 
 @each $state in map-keys($galaxy-state-border) {
     .state-color-#{$state},
     [data-state="#{$state}"] {
         border-color: map-get($galaxy-state-border, $state);
-        background-color: map-get($galaxy-state-bg, $state);
+        background-color: change-color(map-get($galaxy-state-bg, $state), $alpha: 0.4, $lightness: 50%);
+
+        &:hover {
+            background-color: change-color(map-get($galaxy-state-bg, $state), $alpha: 0.4, $lightness: 60%);
+        }
 
         &:focus,
         &:focus-within {
```

Comment:
In dev, focused and hovered elements are slightly lighter than only focused elements, which is a state currently not covered by this change.